<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMain</class>
 <widget class="QMainWindow" name="frmMain">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>485</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>735</width>
    <height>485</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>739</width>
    <height>485</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Генератор сертификатов</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/icon.png</normaloff>../img/icon.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>40</y>
      <width>730</width>
      <height>390</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::South</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="step_0">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="icon">
      <iconset>
       <normaloff>../img/0.png</normaloff>../img/0.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Шаг</string>
     </attribute>
     <attribute name="toolTip">
      <string>Мастер создания сертификатов</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>269</width>
        <height>300</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../img/icon.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>411</width>
        <height>241</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;h2 style=&quot; margin-top:16px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:x-large; font-weight:600;&quot;&gt;Добро пожаловать в мастер создания сертификатов &amp;quot;&lt;/span&gt;&lt;span style=&quot; font-size:x-large; font-weight:600; color:#ef2929;&quot;&gt;SertGen&lt;/span&gt;&lt;span style=&quot; font-size:x-large; font-weight:600;&quot;&gt;&amp;quot; &lt;/span&gt;&lt;/h2&gt;&lt;h3 style=&quot; margin-top:14px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:large; font-weight:600;&quot;&gt;С помощью этой программы вы можете массово подготовить электронные сертификаты в формате PDF для участников различных мероприятий.&lt;/span&gt;&lt;/h3&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblInkscape">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>310</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Путь к редактору Inkscape</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtInkscape">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>330</y>
        <width>391</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>d:\progs\InkscapePortable\InkscapePortable.exe</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblInkscapeIcon">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>330</y>
        <width>24</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../img/check.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="step_1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="icon">
      <iconset>
       <normaloff>../img/1.png</normaloff>../img/1.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Шаг</string>
     </attribute>
     <attribute name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Шаг 1: Выбор шаблона сертификата&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </attribute>
     <widget class="QLabel" name="lblSVG">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>450</width>
        <height>320</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>Внешний вид сертификата</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../img/noimage.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtFileName">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>380</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>1.svg</string>
      </property>
     </widget>
     <widget class="QListWidget" name="lstSVGFiles">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>30</y>
        <width>250</width>
        <height>280</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Просмотр шаблона</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Файл шаблона</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOpenTemplatesFolder">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>320</y>
        <width>250</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Открыть папку с шаблонами</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="step_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="icon">
      <iconset>
       <normaloff>../img/2.png</normaloff>../img/2.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Шаг</string>
     </attribute>
     <attribute name="toolTip">
      <string>Шаг 2: Загрузка данных</string>
     </attribute>
     <widget class="QListWidget" name="lstXLSFiles">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>30</y>
        <width>250</width>
        <height>280</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>450</width>
        <height>320</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnToCSV">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>380</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Таблица</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Файл</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOpenXLSFolder">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>320</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Открыть папку с таблицами</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="step_3">
     <attribute name="icon">
      <iconset>
       <normaloff>../img/3.png</normaloff>../img/3.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Шаг</string>
     </attribute>
     <attribute name="toolTip">
      <string>Шаг 3: Создание</string>
     </attribute>
     <widget class="QPushButton" name="btnMake_all_sert">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>200</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Создать</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>711</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="lblAnimation">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>355</width>
        <height>170</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;h1 align=&quot;center&quot; style=&quot; margin-top:18px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:xx-large; font-weight:600;&quot;&gt;Почти всё готово!&lt;/span&gt;&lt;/h1&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Нажмите кнопку &lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Создать&lt;/span&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt; для запуска процесса создания сертификатов для участия вашего мероприятия&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOpenResults">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>270</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Открыть папку с сертификатами</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtLog">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>340</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>331</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Отчёт</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnNext">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>410</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Вперёд</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string extracomment="апрапр">Вперёд </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>410</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>720</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;h1 style=&quot; margin-top:18px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:xx-large; font-weight:600;&quot;&gt;Мастер создания сертификатов&lt;/span&gt;&lt;/h1&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHelp">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>410</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Справка</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpenDB">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>5</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть базу</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>739</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
